src/compiler/program.ts

```diff
@@ -12,6 +12,12 @@ namespace ts {
 
     const emptyArray: any[] = [];
 
+    const defaultLibrarySearchPaths = <Path[]>[
+        "types/",
+        "node_modules/",

```

Comment:
cast to `Path` - yes, I'll drop it. As for `defaultLibrarySearchPaths ` - no, they are still used [here](https://github.com/Microsoft/TypeScript/pull/7775/files#diff-08a3cc4f1f9a51dbb468c2810f5229d3R239)